packages/theme-cart/request.js

```diff
@@ -36,6 +36,14 @@ export function cartAdd(id, quantity, properties) {
   return fetchJSON('/cart/add.js', config);
 }
 
+export function cartAddFromForm(formData) {
+  var config = {};
```

Comment:
I know we talked about this yesterday. Should this include the headers/credentials?

```
credentials: 'same-origin',
      headers: {
        'X-Requested-With': 'XMLHttpRequest',
        'Content-Type': 'multipart/form-data;'
      }
```

Reply:
For this, I kept the default object with `credentials` and `X-Requested-With`. 

`Content-Type` must be removed for `FormData` elements.
https://muffinman.io/uploading-files-using-fetch-multipart-form-data/